arrays.js

```diff
@@ -0,0 +1,276 @@
+// Textarea Element
+const resultDiv = document.getElementById("resultDiv");
+
+// Using Custom Function intead of console.log
+function logger(parm) {
+  // This function will print result in textarea
+  resultDiv.value = [...parm];
+}
+
+// Write a JavaScript program to compute the sum of three elements of a given array of integers of length 3.
+function sumOfThreeElements() {
+  const givenData = [1, 2, 4];
+  let answer;
+  if (givenData.length == 3) {
+    answer = givenData.reduce((sum, current) => sum + current, 0);
+  }
+  logger(["input Array", givenData, "sumOfThreeElements", answer]);
+}
+
+// Write a JavaScript program to rotate the elements left in a given array of integers of length 3.
+function rotateArrayLeft() {
+  const givenData = [1, 2, 4];
+  if (givenData.length == 3) {
+    givenData.push(givenData.shift());
+  }
+  logger(["Input Array", [1, 2, 4], "rotateArrayLeft", givenData]);
+}
+
+// Write a JavaScript program to check whether 1 appears in the first or last position of a given array of integers. The array length must be larger than or equal to 1.
+function lastOrFirst() {
+  const givenData = [1, 2, 4];
+  let answer = false; // Default Value
+  if (givenData[0] === 1 || givenData[givenData.length] === 1) {
+    answer = true;
+  }
+  logger(["input Array", givenData, "lastOrFirst", answer]);
+}
+
+// Write a JavaScript program to reverse the elements of a given array of integers of length 3.
+function reverseArray() {
+  const array = [1, 2, 4];
+  let answer;
+  answer = array.reverse();
+  logger(["input Array", array, "reverseArray", answer]);
+}
+
+// Write a JavaScript program to create an array by taking the first and last elements from a given array of integers. The length must be larger than or equal to 1.
+function createByFirstAndLast() {
+  const array = [1, 2, 4];
+  let answer;
+  if (array.length >= 1) {
+    answer = [array[0], array[array.length - 1]];
+  }
+  logger(["input Array", array, "createByFirstAndLast", answer]);
+}
+
+// Write a JavaScript program to find the longest string from a given array of strings.
+function findLongestString() {
+  const givenStrings = ["abcdefghijk", "abcde", "abcdefghij"];
+  let stringLength = 0;
+  let stringName = "";
+  givenStrings.forEach((item) => {
+    if (stringLength < item.length) {
+      stringLength = item.length;
+      stringName = item;
+    }
+  });
+  logger([
+    "input Array",
+    givenStrings,
+    "findLongestString",
+    `The Longest String is ${stringName} which has length of ${stringLength}cm`,
+  ]);
+}
+
+// Write a JavaScript program to divide a given array of positive integers into two parts. First element belongs to the first part, second element belongs to the second part, and third element belongs to the first part and so on. Now compute the sum of two parts and store it in an array of size two.
+function divideAndSum() {
+  const arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
```

Comment:
We should change the name of the variable.

Reply:
Yes, I changed that on pull request 3, That is Yesterday